src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -95,7 +95,15 @@ if [ "$(uname -s)" == "Darwin" ]; then
 elif [ "$(uname -s)" == "Linux" ]; then
   # On Linux, we'll enable core file generation unconditionally, and if a dump
   # is generated, we will print some useful information from it and delete the
-  # dump immediately. 
+  # dump immediately.
+
+  if [ -e /proc/self/coredump_filter ]; then
```

Comment:
Is there a better place to do this? I want to try and keep these test scripts to a minimum and avoid most of these sort of hacks.